app/src/renderer/components/staking/PageBond.vue

```diff
@@ -286,6 +286,7 @@ export default {
     bondBarPercent(dividend) {
       let divisor = this.totalAtoms
       let ratio = Math.round(dividend / divisor * 100)
+      if (isNaN(ratio)) ratio = 0
```

Comment:
This is OK but next time it's better to write like this:

```
if (isNan(ratio)) { ratio = 0 }
```

That calls out more clearly that there's a side effect being caused.

Reply:
👍